azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/handlers/AnnotationHandlerImplTest.java

```diff
@@ -87,7 +96,33 @@
     public static final String[] EVENTHUB_TRIGGER_REQUIRED_ATTRIBUTES = new String[]{"name", "type", "direction",
         "connection", "eventHubName", "cardinality", "consumerGroup"};
 
-    public class FunctionEntryPoints {
+    public static final String[] CUSTOM_BINDING_REQUIRED_ATTRIBUTES = new String[]{"name", "type", "direction"};
+
+    public static final String[] EXTENDING_CUSTOM_BINDING_REQUIRED_ATTRIBUTES = new String[]{"name", "type",
+        "direction", "path"};
+
+    public static class FunctionEntryPoints {
+
+        @Target(ElementType.PARAMETER)
+        @Retention(RetentionPolicy.RUNTIME)
+        @CustomBinding(direction = "in", name = "message", type = "customBinding")
+        public @interface TestCustomBinding {
+            String direction();
+            String path();
+            String name();
+        }
+
+        @FunctionName(CUSTOM_BINDING_FUNCTION)
+        public void customBindingMethod(
```

Comment:
Add one more test where TestCustomBinding does not define a name.
```java
@FunctionName(EXTENDING_CUSTOM_BINDING_FUNCTION)
        public void extendingcustomBindingMethod2(
            @TestCustomBinding(direction = "out",  path = "testPath")
                    String customTriggerInput) {
        }
```
In this case, name should be taken from customBinding: message